tests/otel_integrations/test_sqlalchemy.py

```diff
@@ -207,6 +208,182 @@ class AuthRecord(Base):
+                    'logfire.msg': 'select * from auth_records',
+                    'db.statement': 'select * from auth_records',
+                    'db.system': 'sqlite',
+                    'db.name': 'example.db',
+                },
+            },
+            {
+                'name': 'INSERT example.db',
+                'context': {'trace_id': 7, 'span_id': 13, 'is_remote': False},
+                'parent': None,
+                'start_time': 13000000000,
+                'end_time': 14000000000,
+                'attributes': {
+                    'logfire.span_type': 'span',
+                    'logfire.msg': 'INSERT INTO auth_records (id, number, content) VALUES (?, ?, ?)',
+                    'db.statement': 'INSERT INTO auth_records (id, number, content) VALUES (?, ?, ?)',
+                    'db.system': 'sqlite',
+                    'db.name': 'example.db',
+                },
+            },
+            {
+                'name': 'connect',
+                'context': {'trace_id': 8, 'span_id': 15, 'is_remote': False},
+                'parent': None,
+                'start_time': 15000000000,
+                'end_time': 16000000000,
+                'attributes': {
+                    'logfire.span_type': 'span',
+                    'logfire.msg': 'connect',
+                    'db.name': 'example.db',
+                    'db.system': 'sqlite',
+                    'logfire.level_num': 5,
+                },
+            },
+            {
+                'name': 'SELECT example.db',
+                'context': {'trace_id': 9, 'span_id': 17, 'is_remote': False},
+                'parent': None,
+                'start_time': 17000000000,
+                'end_time': 18000000000,
+                'attributes': {
+                    'logfire.span_type': 'span',
+                    'logfire.msg': 'SELECT auth_recor…ds_content FROM auth_records WHERE …',
+                    'db.statement': """\
+SELECT auth_records.id AS auth_records_id, auth_records.number AS auth_records_number, auth_records.content AS auth_records_content \nFROM auth_records \nWHERE auth_records.id = ?\
+""",
+                    'db.system': 'sqlite',
+                    'db.name': 'example.db',
+                },
+            },
+            {
+                'name': 'DELETE example.db',
+                'context': {'trace_id': 10, 'span_id': 19, 'is_remote': False},
+                'parent': None,
+                'start_time': 19000000000,
+                'end_time': 20000000000,
+                'attributes': {
+                    'logfire.span_type': 'span',
+                    'logfire.msg': 'DELETE FROM auth_records WHERE auth_records.id = ?',
+                    'db.statement': 'DELETE FROM auth_records WHERE auth_records.id = ?',
+                    'db.system': 'sqlite',
+                    'db.name': 'example.db',
+                },
+            },
+        ]
+    )
+
+    SQLAlchemyInstrumentor().uninstrument()
```

Comment:
```suggestion
```

Reply:
We can't, bug in OTel regarding uninstrumentation/instrumentation.